operator/pkg/apis/cassandra/v1beta1/cassandradatacenter_types.go

```diff
@@ -181,6 +181,9 @@ type CassandraDatacenterSpec struct {
 
 	// Container image for the log tailing sidecar container.
 	SystemLoggerImage string `json:"systemLoggerImage,omitempty"`
+
+	// AdditionalServiceConfig allows to define additional parameters that are included in the created Services
```

Comment:
Can we add a warning blurb here to let the user know he can break things by overwriting labels set by cass-operator so extra caution should take when setting this property.

Reply:
Ah, it doesn't copy it from the ServiceConfig description.